### ISSUE TITLE
Tick down remaining kills on each mission.

### ISSUE THREAD

**Issue by joncage** [lunarplasma/EDMC-CombatTracker#20]
Would be neater to tick down the kills for each mission as you kill pirates.

Not easy but maybe if you know the target faction and the time the mission was accepted, you could tick them off one by one.

**Comment by lunarplasma** [comment]
Yeah, I had considered doing it as an estimate. Maybe even make it so that it's a textbox that the user can manually correct if they so wished.

**Comment by Sebyujii** [comment]
I'm not too sure about the implementation but I wouldnt recommend checking the currently targeted ships and count +1 whenever I retarget another pirate. Most of the time I would start attacking all pirates of a wing and afterwards kill them one by one. With the thought above, that could maybe count every pirate two times.

**Comment by joncage** [comment]
I was thinking each time a bounty is paid (caveat; I've not checked the logs to see how feasible this is), use that to tick down the number. 

Doing it when you target a pirate wouldn't be enough I don't think - lots of ways that could go wrong.

**Comment by Sebyujii** [comment]
yeah problem being, you probably are not only killing mission targets in res, right?

**Comment by jberanek** [comment]
Well, an example journal entry:

`{ "timestamp":"2021-04-21T18:24:46Z", "event":"Bounty", "Rewards":[ { "Faction":"LHS 1358 Fortune Group", "Reward":70245 } ], "Target":"federation_dropship", "Target_Localised":"Federal Dropship", "TotalReward":70245, "VictimFaction":"LHS 1358 Jet Gang", "SharedWithOthers":2 }`

**Comment by jberanek** [comment]
However, you don't know which mission(s) that will apply to.

**Comment by lunarplasma** [comment]
> However, you don't know which mission(s) that will apply to.

It should apply to each top mission on the stack for that faction - ie. the oldest ones.

Yeah, that bounty event looks promising.

**Comment by joncage** [comment]
> It should apply to each top mission on the stack for that faction - ie. the oldest ones.

That was my thinking.
